app/services/force/soql_query_builder.rb

```diff
@@ -13,8 +13,8 @@ def initialize(client)
     # BUILDER
     ################################
 
-    def select(select)
-      @select = select
+    def select(*select)
```

Comment:
what motivated the changes in this file?

Reply:
I did it this way an array as arguments like `select(:a,:b,:c)` instead of `select(i%{a b c})` or `select([:a, :b, :c])`